src/components/DocumentationTopic/PrimaryContent.vue

```diff
@@ -152,7 +152,9 @@ export default {
     content: '';
     display: block;
   }
+}
 
+.primary-content:not(.minimized-content) {
```

Comment:
There should be a way to eliminate the need for the addition of this `:not()` selector based on how you structure the overriding rule in `DocumentationTopic.vue`. Maybe you tried this at first and it didn't work? If not, maybe you weren't overriding it in the same way it's defined here?